jupyter/jupyter-web-app/overlays/application/application.yaml

```diff
@@ -1,16 +1,11 @@
 apiVersion: app.k8s.io/v1beta1
 kind: Application
 metadata:
-  name: jupyter-web-app
+  name: $(generateName)
 spec:
   selector:
     matchLabels:
-      app.kubernetes.io/name: jupyter-web-app
-      app.kubernetes.io/instance: jupyter-web-app
-      app.kubernetes.io/managed-by: kfctl
-      app.kubernetes.io/component: jupyter
-      app.kubernetes.io/part-of: kubeflow
-      app.kubernetes.io/version: v0.6
+      app.kubernetes.io/instance: $(generateName)
```

Comment:
@kkasravi What's the reasoning for introducing generateName here? The selector was already matching a set of labels that was already being consistently applied across all the components via kustomization.yaml commonLabels.

Why are we parameterizing this using generateName?